### DISCUSSION TITLE
play jingles

### DISCUSSION THREAD

**Discussion by sander816** [AzuraCast/AzuraCast#5722] [General]
i added a playlist for our station jingles, as per advanced playlist options it shoudl play once every 15 minutes
But everytime it plays all jingles??

**Comment by Mozitux** [comment]
The jingles must each be in specific playlists in order to apply independent time constraints to them.

**Comment by sander816** [comment]
Sorry my fault. Jingles were in one single file.
Now it's playing ok

But, whenever a jingle is loaded to play after a song, theres a short interruption in the stream. This doesn't happen when songs are playing 

And the crossfade is not working with the 5 second jingles. On songs it's ok, with the jingles there's sometimes seconds silence before the jingle plays
